pkg/rakkess/util/printer.go

```diff
@@ -100,3 +114,18 @@ func codeToColor(code int) color {
 	}
 	return none
 }
+
+func asciiAccessCode(code int) string {
+	switch code {
+	case client.AccessAllowed:
+		return "yes"
+	case client.AccessDenied:
+		return "no"
+	case client.AccessNotApplicable:
+		return ""
```

Comment:
As you seem to have in mind to use ascii-mode in scripts: it would greatly ease parsability if "not applicable" would not print as the empty string, but "n/a" or so. What do you think?